conformance/reports/v1.3.0/kgateway/experimental-main-report.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: gateway.networking.k8s.io/v1
+date: "2025-06-18T13:02:08-04:00"
+gatewayAPIChannel: experimental
+gatewayAPIVersion: v1.3.0
+implementation:
+  contact:
+  - github.com/kgateway-dev/kgateway/issues/new/choose
+  organization: kgateway-dev
+  project: kgateway
+  url: github.com/kgateway-dev/kgateway
+  version: 1.0.1-dev
```

Comment:
Is this a released version that can be used? If not, is it possible to run these tests against a released version of the controller?

Reply:
Yeah, I think that's fine. We always prefer a report against a released version, but it's not a blocker.

Reply:
SG and makes sense. For posterity, we had some reports of user confusion when other implementations published conformance results against main/latest version, and this seemed like the easiest approach in the short term.